specifyweb/frontend/js_src/lib/localization/wbPlan.ts

```diff
@@ -418,6 +418,9 @@ export const wbPlanText = createDictionary({
     'uk-ua': 'Показати додаткові таблиці',
     'de-ch': 'Erweiterte Tabellen anzeigen',
   },
+  showAdvancedTables: {
```

Comment:
we already have `wbText.showAllTables()`. is this slightly different string added intentionally?
can we change them both to use the same label?